### ISSUE TITLE
Fading a chart in/out using Chart.js and ngIf/ngAnimate doesn't work - chart animates in but not out

### ISSUE THREAD

**Issue by nfriend** [jtblin/angular-chart.js#525]
### Overview

Describe the issue. What is the issue and what did you expect?

I have a Chart.js instance which I'd like to fade in/out using ngIf/ngAnimate.  When fading in, the chart animates from `opacity: 0` to `opacity: 1` as expected.  However, when going the other direction, the chart disappears immediately instead of animating.  See the jsfiddle below for a demo of this behavior.  Note that other element within the ngIf-ed container animate properly in both directions.

Please make sure to review and check all of these items:

- [ x ] Use latest version of the library
 - using `angular-chart.js` 1.0.3, `Chart.js` 2.3.0, and `angular` 1.5.8
- [ x ] Make sure you've included all the dependencies e.g Chart.js, angular, etc.
- [ x ] Include a repro case, see below.

### Step to reproduce

See a demo here: https://jsfiddle.net/NathanFriend/p48tqupn/

**Comment by hans0low** [comment]
@nfriend did you ever find a solution to this?

**Comment by nfriend** [comment]
@hans0low I don't think so, but it's been so long ago I actually don't remember! Sorry!

**Comment by jtblin** [comment]
This is a usage question or stale support request from several years ago. As we are currently modernizing the library to support Chart.js v4 and TypeScript, we are closing stale issues. If this is still a problem in the latest version, please open a new issue with a modern reproduction.
